sos/plugins/services.py

```diff
@@ -9,26 +9,23 @@
 from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class Services(Plugin):
+class Services(Plugin, DebianPlugin, UbuntuPlugin):
     """System services
     """
 
     plugin_name = "services"
     profiles = ('system', 'boot')
 
-    option_list = [("servicestatus", "get a status of all running services",
-                    "slow", False)]
-
```

Comment:
Didn's see any rationale for dropping the option here? In some ways, it's moot, since most distros have moved to Systemd anyway and use that plugin for this information but we shouldn't remove options without good reason (they break automation scripts that assume the option exists).

Reply:
We default to collecting the equivalent for systems without putting it behind a flag.